tests/e2e/parallel/main_test.go

```diff
@@ -2138,72 +2138,211 @@ func TestScanSettingBinding(t *testing.T) {
 
 }
 
-func TestScanSettingBindingTailoringAndNonDefaultRole(t *testing.T) {
+func TestScanSettingBindingTailoringManyEnablingRulePass(t *testing.T) {
+
 	t.Parallel()
 	f := framework.Global
-	tpName := "non-default-role-tp"
-	scanSettingBindingName := "non-default-role-ssb"
+	const (
+		changeTypeRule      = "kubelet-anonymous-auth"
+		unChangedTypeRule   = "api-server-insecure-port"
+		moderateProfileName = "moderate"
+		tpMixName           = "many-migrated-mix-tp"
+		tpSingleName        = "migrated-single-tp"
+	)
+	var (
+		baselineImage = fmt.Sprintf("%s:%s", brokenContentImagePath, "kubelet_default")
+		modifiedImage = fmt.Sprintf("%s:%s", brokenContentImagePath, "new_kubeletconfig")
+	)
 
-	tp := &compv1alpha1.TailoredProfile{
+	prefixName := func(profName, ruleBaseName string) string { return profName + "-" + ruleBaseName }
+
+	pbName := framework.GetObjNameFromTest(t)
+	origPb := &compv1alpha1.ProfileBundle{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      tpName,
+			Name:      pbName,
+			Namespace: f.OperatorNamespace,
+		},
+		Spec: compv1alpha1.ProfileBundleSpec{
+			ContentImage: baselineImage,
+			ContentFile:  framework.OcpContentFile,
+		},
+	}
+	// Pass nil in as the cleanupOptions since so we don't invoke all the
+	// cleanup function code in Create. Use defer to cleanup the
+	// ProfileBundle at the end of the test, instead of at the end of the
+	// suite.
+	if err := f.Client.Create(context.TODO(), origPb, nil); err != nil {
+		t.Fatalf("failed to create ProfileBundle: %s", err)
+	}
+	// This should get cleaned up at the end of the test
+	defer f.Client.Delete(context.TODO(), origPb)
+
+	if err := f.WaitForProfileBundleStatus(pbName, compv1alpha1.DataStreamValid); err != nil {
+		t.Fatalf("failed waiting for the ProfileBundle to become available: %s", err)
+	}
+
+	// Check that the rule exists in the original profile
+	changeTypeRuleName := prefixName(pbName, changeTypeRule)
+	err, found := f.DoesRuleExist(origPb.Namespace, changeTypeRuleName)
+	if err != nil {
+		t.Fatal(err)
+	} else if found != true {
+		t.Fatalf("expected rule %s to exist in namespace %s", changeTypeRuleName, origPb.Namespace)
+	}
+
+	// Check that the rule exists in the original profile
+	unChangedTypeRuleName := prefixName(pbName, unChangedTypeRule)
+	err, found = f.DoesRuleExist(origPb.Namespace, unChangedTypeRuleName)
```

Comment:
Since `api-server-insecure-port` is included in the default ocp4 profile bundle, do we know if this is passing because it's being used there? Should the assertion check the profile the rule came from, too?

Reply:
So the tests are first asserting two rules are already in the profile bundle ocp4:
```
		changeTypeRule      = "kubelet-anonymous-auth"
		unChangedTypeRule   = "api-server-insecure-port"
```
Then we update the profile bundle that changes the type of `kubelet-anonymous-auth` to a different check type.